physionet-django/events/templates/events/event_pending_applications.html

```diff
@@ -0,0 +1,31 @@
+{% load participation_status %}
+<div class="tab-content" id="nav-tabContent">
+  <div class="tab-pane fade show active" id="active-participant" role="tabpanel" aria-labelledby="active-participant-tab">
```

Comment:
In both those places the id will not be unique (since you are rendering all 3 templates, you'll have 3 divs with the "active-participant" id for example).

Also - this should also be a single template and the only configurable part would be the list of participants. I see that all the templates use the same table body:
```html
<td>{{ application.user.username }}</td>
<td>{{ application.user.get_full_name }}</td>
<td>{{ application.user.email }}</td>
<td>{{ application.comment_to_applicant }}</td>
<td>{{ application.decision_datetime | date }}</td>
```

Is there any other difference? Either way be can make it configurable with arguments and have one template that renders the list of applications.